casl-requirements.yml

```diff
@@ -0,0 +1,8 @@
+
+# This is the Ansible Galaxy requirements file to pull in the correct roles
+# to support the operation of CASL provisioning/runs. 
+
+# From 'infra-ansible'
+- src: https://github.com/redhat-cop/infra-ansible
```

Comment:
Although the command ``ansible-galaxy install git+https://github.com/oybed/infra-ansible,galaxy -p roles`` works